example/sync/main.go

```diff
@@ -62,7 +62,7 @@ func run() (err error) {
 				return err
 			}
 
-			fmt.Println("%d frames synced", replicated.FramesSynced)
+			fmt.Printf("%d frames synced", replicated.FramesSynced)
```

Comment:
Why would this be a good idea?

Reply:
Good catch. You missed the new line though, no?

Reply:
updated.